transform/variants/variants_test.go

```diff
@@ -1,39 +1,95 @@
-package variants_test
+package variants
 
 import (
 	"fmt"
-	"log"
 	"sort"
 	"testing"
 
-	"github.com/TimothyStiles/poly/transform/variants"
+	"github.com/stretchr/testify/assert"
```

Comment:
Increases dependencies. However, it seems like assert is a pretty common way for people to test in Go, and also is a little bit more concise than just using an `==` + `t.Errorf`. Thoughts about using `assert` for wider testing @TimothyStiles ?

If it is the case that we want to use asserts more widely, we should keep it here, otherwise, we should go with the t.Errorf basic method to keep consistent with the rest of the package.

Reply:
I think testify is already a dependency, used by the codon tests?

Reply:
That's a good point (should have checked go.mod!) Didn't check that when the PR went thru https://github.com/TimothyStiles/poly/pull/232

Reply:
> I think testify is already a dependency, used by the codon tests?

This is correct @fergloragain. @Koeng101 it was more concise and I thought it may be good to migrate towards using testify everywhere. Maybe we should make this another issue?